website/.docusaurus/DONT-EDIT-THIS-FOLDER

```diff
@@ -0,0 +1,5 @@
+This folder stores temp files that Docusaurus' client bundler accesses.
```

Comment:
All of the website/.docusaurus files should be deleted from this PR. I assume they were already committed before they were added to the .gitignore file

Reply:
good catch, removed it all